invenio_drafts_resources/services/draft_service.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+
+"""Draft Service."""
+
+from invenio_records_resources.resource_unit import IdentifiedRecord
+from invenio_records_resources.services import RecordService, \
+    RecordServiceConfig
+
+from ..resource_unit import IdentifiedDraft
+from .permissions import DraftPermissionPolicy
+
+
+class DraftServiceConfig(RecordServiceConfig):
+    """Draft Service configuration."""
+
+    # Service configuration
+    permission_policy_cls = DraftPermissionPolicy
+    resource_unit_cls = IdentifiedDraft
+
+    # RecordService configuration
+    # TODO: FILL ME!
+
+    # DraftService configuration
+    # TODO: FILL ME!
+    draft_of_resource_unit_cls = IdentifiedRecord
```

Comment:
I think this might need to be a service class/instance though. e.g. When creating a draft of a non-existing record (new) it will need to create the record + draft (and associate them). Therefore, it would be great to use the service so data validation etc. happens already in the background. For now can be shelved until we have this more clear (just putting my thoughts here)

Reply:
After sleeping on it, I was even wondering if we could just call the `RecordService::create()` method in `publish`, this way that configuration isn't really needed. When implementing this can be changed to something that is more appropriate, so we can wait until that happens.